src/components/shared/TransactionSelectInput/index.jsx

```diff
@@ -54,69 +57,87 @@ function TransactionSelectInput({
-        {labelIcon}
-        <LabelText>{label}:</LabelText>
-      </InputLabel>
-      <Controller
-        name={name}
-        control={control}
-        render={({ field: { onChange, value } }) => (
-          <Select
-            components={{ NoOptionsMessage: MenuInviteMessage }}
-            hideSelectedOptions={false}
-            options={options}
-            placeholder={placeholder}
-            value={value}
-            onChange={onChange}
-            onInputChange={(val, { action }) => {
-              handleInputChange(val, name);
-              // prevent clear input when not focused
-              if (action !== 'set-value' && val) {
-                setValue(name, { id: val, name: val });
-                setModalData((prev) => ({
-                  ...prev,
-                  type: label,
-                  val,
-                }));
-              }
-            }}
-            className="transaction-select"
-            classNamePrefix="transaction"
-            theme={(theme) => customSelectTheme(theme, error)}
-            isClearable
-            isSearchable
-            getOptionLabel={(option) => option.name}
-            getOptionValue={(option) => option.id}
-            type={label}
-          />
+    <div>
+      <SelectContainer noOptions={options.length > 0} rounded={rounded}>
+        {label && (
+          <InputLabel>
+            {labelIcon}
+            <LabelText>{label}:</LabelText>
+          </InputLabel>
         )}
-      />
-    </SelectContainer>
+        <Controller
+          name={name}
+          control={control}
+          render={({ field: { onChange, value } }) => (
+            <Select
+              components={
+                forInvite ? { NoOptionsMessage: MenuInviteMessage } : {}
+              }
+              hideSelectedOptions={false}
+              options={options}
+              placeholder={placeholder}
+              value={value}
+              onChange={onChange}
+              onInputChange={(val, { action }) => {
+                if (forInvite) {
+                  handleInputChange(val, name);
+                  // prevent clear input when not focused
+                  if (action !== 'set-value' && val) {
+                    setValue(name, { id: val, name: val });
+                    setModalData((prev) => ({
+                      ...prev,
+                      type: label,
+                      val,
+                    }));
+                  }
+                }
+              }}
+              className="transaction-select"
+              classNamePrefix="transaction"
+              theme={(theme) => customSelectTheme(theme, error)}
+              isClearable
+              isSearchable
+              getOptionLabel={(option) => option.name}
+              getOptionValue={(option) => option.id}
+              type={label}
+            />
+          )}
+        />
+      </SelectContainer>
+      {control && <ErrorMessage>{error || ''}</ErrorMessage>}
```

Comment:
do we need to render the error if there is no error
```suggestion
      {error && <ErrorMessage>{error}</ErrorMessage>}
```

Reply:
i need it so the flex layout won't corrupt when the error appears